packages/component-library/src/NotebookPreview/NotebookPreview.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from "prop-types";
+import Card from "@material-ui/core/Card";
+import { makeStyles } from "@material-ui/core/styles";
+
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+
+import Icon from "../Icon/Icon";
+
+const header = css`
+  padding-left: 1rem;
+`;
+
+const useProjectStyles = makeStyles({
+  card: {
+    "&:hover": {
+      boxShadow:
+        "0px 2px 6px 0px rgba(0,0,0,0.2), 0px 2px 2px 0px rgba(0,0,0,0.14), 0px 4px 2px -1px rgba(0,0,0,0.12)"
+    }
+  }
+});
+
+const NotebookPreview = ({ message, link }) => {
+  const classes = useProjectStyles();
+
+  return (
+    <a href={link}>
```

Comment:
I think this would be better with a `target="_blank"` so that it opens in a new tab